components/brave_wallet/browser/swap_response_parser.cc

```diff
@@ -89,6 +104,16 @@ mojom::SwapResponsePtr ParseSwapResponse(const std::string& json,
       swap_response_value->sell_token_to_eth_rate;
   swap_response->buy_token_to_eth_rate =
       swap_response_value->buy_token_to_eth_rate;
+  swap_response->estimated_price_impact =
+      swap_response_value->estimated_price_impact;
+
+  for (const auto& source_value : swap_response_value->sources) {
+    mojom::ZeroExSource source;
```

Comment:
nit: mojom::ZeroExSource::New() creates a ZeroExSourcePtr which could be moved into vector so no need in cloning.

Reply:
Actually could be inlined as push_back(mojom::ZeroExSource::New(source_value.name, source_value.proportion))

Reply:
Fixed https://github.com/brave/brave-core/pull/15762/commits/c4d08e374d30735c892cecaeeeb0f6b5c68bcad9.